### PR TITLE
Updated DJED contract address

### PR DESCRIPTION
**Pull request by a-singh09**

### CODE REVIEW

File: env/ethereum-classic.env [hunk 1]
```diff
@@ -3,10 +3,10 @@
 ###############
 REACT_APP_BLOCKCHAIN_URI=https://etc.rivet.link
 REACT_APP_CHAIN_ID=61
-REACT_APP_DJED_ADDRESS=0x847CceD1eF8Dde95FaB8Eb11B281aAA23Dd014D6
+REACT_APP_DJED_ADDRESS=0x621e39a1A4e64f6C3279e55d6EAc97B7Ee8A37b5
 REACT_APP_FEE_UI=0
 REACT_APP_UI=0x0249a1a7778BD203206d1B7b614fB35Ce7f8F285
-REACT_APP_EXPLORER=https://blockscout.com/etc/mainnet/
+REACT_APP_EXPLORER=https://etc.blockscout.com/
```
Comment (ceilican) [inline env/ethereum-classic.env:9]: Could you change this line to:

`REACT_APP_EXPLORER=https://etc.blockscout.com`

Comment (ceilican) [reply env/ethereum-classic.env:9]: Otherwise the last slash will cause problems in the address.

**Review by ceilican — CHANGES_REQUESTED** [review]

File: src/routes/stablecoin.jsx [hunk 2]
```diff
@@ -322,7 +322,7 @@ export default function Stablecoin() {
               StableCoins are implemented as a standard ERC-20 token contract and the
               contract's address is{" "}
               <a
-                href={`${process.env.REACT_APP_MILKOMEDA_C1_EXPLORER}address/${coinContracts?.stableCoin._address}`}
+                href={`${process.env.REACT_APP_EXPLORER}/address/${coinContracts?.stableCoin._address}`}
```
Comment (ceilican) [inline src/routes/stablecoin.jsx:325]: or maybe remove the slash from here...

File: src/routes/reservecoin.jsx [hunk 3]
```diff
@@ -335,7 +335,7 @@ export default function ReserveCoin() {
               ReserveCoins are implemented as a standard ERC-20 token contract and the
               contract's address is{" "}
               <a
-                href={`${process.env.REACT_APP_MILKOMEDA_C1_EXPLORER}address/${coinContracts?.reserveCoin._address}`}
+                href={`${process.env.REACT_APP_EXPLORER}/address/${coinContracts?.reserveCoin._address}`}
```
Comment (ceilican) [inline src/routes/reservecoin.jsx:338]: ...and here.

**Review by ceilican — CHANGES_REQUESTED** [review]

File: env/ethereum-classic.env (hunk 1 above)
Comment (ceilican) [reply env/ethereum-classic.env:9]: See my other comments.

Whatever you decide to do, ensure that it works with the other env files too (for milkomeda, sepolia, ...).

Comment (a-singh09) [reply env/ethereum-classic.env:9]: Sure, I have fixed it. Somehow despite the slash, the browser seems to correctly open it, so couldn't detect that.

**Pull request merged**